infrastructure/cinco/config/stage/arclight/build.yaml

```diff
@@ -32,4 +32,17 @@ sceptre_user_data:
                     - docker push ${AWS::AccountId}.dkr.ecr.us-west-2.amazonaws.com/cinco-arclight:latest
                 post_build:
                   commands:
-                    - aws ecs update-service --cluster cinco-stage --service cinco-arclight-stage-service --force-new-deployment
```

Comment:
I think we still want to force the new deployment after build, right?

Reply:
Yes! Thanks for catching that